index.html

```diff
@@ -80,3 +76,221 @@ <h2 class="secondary-header">LET’S CONNECT</h2>
     </section>
   </body>
 </html>
+<!-- WORKS SECTION  -->
+<section class="works">
+  <!-- first project  -->
+ <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio (1).png" alt="" />
+    <h3 class="tertiary-heading">Tonic</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- second project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio.png" alt="" />
+    <h3 class="tertiary-heading">Multi-Post Stories</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- thirs project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio (1).png" alt="" />
+    <h3 class="tertiary-heading">Tonic</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+  <!-- fourth project  -->
+  <div class="projects container">
+    <img class="project-img" src="images/Snapshoot Portfolio.png" alt="" />
+    <h3 class="tertiary-heading">Multi-Post Stories</h3>
+    <ul class="project-details">
+      <li>CONOPY</li>
+      <li>Back End Dev</li>
+      <li>2015</li>
+    </ul>
+    <p class="project-description">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required.
+    </p>
+    <ul class="project-lang">
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+</section>
+<!-- ABOUT ME SECTION  -->
+<section class="about container">
+  <h2 class="secondary-header">About Myself</h2>
+  <p class="headline-description">
+    Hello I’m a software developer! I can help you build a product , feature
```

Comment:
- [ ] I would highly recommend you to add rounded-right corner to your "about container" section to match the [figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=W0MZ6qGGoBjUhLi8-0)

|. Figma design |. You design |
|---------------|-------------|
|<img width="346" alt="Screenshot 2023-04-30 at 3 03 43 AM" src="https://user-images.githubusercontent.com/44798044/235326162-1ccccfd8-10b8-4a12-a9ca-bc7601ba890f.png">|<img width="375" alt="Screenshot 2023-04-30 at 3 02 37 AM" src="https://user-images.githubusercontent.com/44798044/235326177-0d012579-a67a-486f-8fe2-7eea927a6f53.png">|